### PR TITLE
Revise website community content to be more beginner-friendly

### PR DESCRIPTION
**Pull request by evankanderson**
Improves on https://github.com/knative/community/issues/681

## Proposed Changes 

At the Knative Maintainer Meetup at KubeCon NA, it was brought up that it's a little hard for lurkers / prospective community members to get engaged.  During the discussion, it was suggested that we should improve the "contributing" section on the main site to be more focused on new community members who may not have decided to commit to Knative yet (the start of the funnel).

- Rename "contributing" (donating effort) to "community" (participating in some way)
- Improve the "community" landing page based on https://gateway-api.sigs.k8s.io/contributing/ (referenced in discussion)
- Rename "about" to "governance"

### CODE REVIEW

File: docs/community/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# How to Get Involved
+
+<!-- TODO: what is community, what are the touchpoints -->
+
+## Questions
+
+For the fastest response, you can ask questions on the [`#general`, `#serving`
+or `#eventing` channels of the Knative Slack](https://slack.knative.dev/).
```
Comment (psschwei) [inline docs/community/README.md:8]: Should we add `#functions` to this list?

File: docs/community/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+We also have a [Stack Overflow topic,
+`knative`](https://stackoverflow.com/questions/tagged/knative) (as well as
+`knative-serving` and `knative-eventing`), and the
```
Comment (psschwei) [inline docs/community/README.md:12]: same as above

**Review by psschwei — COMMENTED** [review]
I'm good with these changes. 

@abrennan89 was also working on revising this section in #5307, so I wonder how best to proceed? Though based on a _very_ quick glance, I don't think it would be too difficult to reconcile the two.

cc @snneji

File: docs/community/README.md (hunk 1 above)
Comment (evankanderson) [reply docs/community/README.md:31]: Done.

File: docs/community/README.md (hunk 2 above)
Comment (evankanderson) [reply docs/community/README.md:35]: I created the `knative-functions` tag on Stack Overflow just now, so now we can link to its presence

**Comment by evankanderson** [comment]
Merged in #5307 changes and addressed @psschwei 's feedback.

**Comment by evankanderson** [comment]
(This is ready for another look)

File: docs/community/README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+better than a partial report in the right component, so feel free to file your issue in these three main repositories if you're not sure:
+
+* [Serving](https://github.com/knative/serving/issues/new?template=bug-report.md)
+* [Eventing](https://github.com/knative/eventing/issues/new?template=bug.md)
+* [Functions](https://github.com/knative/func/issues/new)
+
+A good bug report should include:
+
+- What you were trying to do, and what happened
+- What version of Knative and Kubernetes you are usinG (if using a cloud provider, indicate which one)
```
Comment (psschwei) [inline docs/community/README.md:55]:
```suggestion
- What version of Knative and Kubernetes you are using (if using a cloud provider, indicate which one)
```

File: docs/community/README.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+[knative-users](https://groups.google.com/forum/#!forum/knative-users) mailing
+list if you prefer those formats.
+
+## Bug Reports and Feature Requests
+
+Knative is composed of many different components. We use GitHub Issues to track
+bug reports and feature requests. While it's most helpful if you know the
+specific component that the bug is happening in, we understand that it can be
+difficult to tell sometimes. A clear bug report in the wrong component is much
+better than a partial report in the right component, so feel free to file your issue in these three main repositories if you're not sure:
```
Comment (psschwei) [inline docs/community/README.md:46]:
```suggestion
better than a partial report in the right component, so feel free to file your issue in one of these three main repositories if you're not sure:
```

File: docs/community/README.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+Drive](https://drive.google.com/drive/folders/0AM-QGZJ-HUA8Uk9PVA). Due to
+limitations of shared drives, the default access control is that all documents
+are *readable* by `knative-users@` and *commentable* and *editable* by
+`knative-dev@`. Documents generally cannot be deleted without special
+intervention.
+
+We also use GitHub Issues and [GitHub
+projects](https://github.com/orgs/knative/projects) for tracking longer-term
+efforts, including each working group's roadmap, and the backlog for oversight
+committees like the Technical Oversight Committee and the Steering Committee.
```
Comment (psschwei) [inline docs/community/README.md:92]: nit: should we move this to just after the slack bit? 

My thinking is that engagement usually goes something like  Slack -> Github -> mailing lists , so might be good to reflect that here

File: docs/community/README.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
+# How to Get Involved
```
Comment (psschwei) [inline docs/community/README.md:1]: nit: the sidebar shows "Welcome to the Knative community" versus "How to Get Involved", should we update the title here?

File: docs/community/governance.md [hunk 7]
```diff
@@ -3,6 +3,8 @@
 This page provides links to documents for common Knative community practices and
```
Comment (psschwei) [inline docs/community/governance.md:3]: same comment here on sidebar vs page title: "Community Rules and Practices" vs "About the community" (github wouldn't let me tag the right line though...)

File: docs/community/contributing.md [hunk 8]
```diff
@@ -85,20 +84,13 @@ or Eventing and chat with other project developers. See also the [Knative Slack
 - [Stack Overflow questions](https://stackoverflow.com/questions/tagged/*knative*):
 Knative tagged questions and curated answers.
 
-## Community Meetup
+### Community Meetups
 
-This virtual event is designed for end users, a space for our community to meet,
-get to know each other, and learn about uses and applications of Knative.
+New to Knative or built something cool?  Share your impressions by
+presenting at the Knative Community Meetup!
```
Comment (psschwei) [inline docs/community/contributing.md:90]: do we have a link on how to do volunteer to present? (coincidentally was looking for this info earlier and couldn't find it)

File: docs/community/contributing.md [hunk 9]
```diff
@@ -85,20 +84,13 @@ or Eventing and chat with other project developers. See also the [Knative Slack
 - [Stack Overflow questions](https://stackoverflow.com/questions/tagged/*knative*):
 Knative tagged questions and curated answers.
 
-## Community Meetup
+### Community Meetups
```
Comment (psschwei) [inline docs/community/contributing.md:87]: Should we keep this as a top level way to contribute vs. under "learn and connect"? (I generally think of them as separate things, but don't have a strong opinion here, so either way is fine)

**Review by psschwei — COMMENTED** [review]
a few suggestions / nits

File: docs/community/contributing.md (hunk 8 above)
Comment (psschwei) [reply docs/community/contributing.md:90]: found it: https://t.co/p7qiU4Dob9

**Comment by psschwei** [comment]
(closing / reopening to pull in the unit tests fix)

File: docs/community/contributing.md (hunk 9 above)
Comment (evankanderson) [reply docs/community/contributing.md:87]: I think the meetups are about learning and connecting, so I think this extra level is logical. (I may also have copied it, I'm not sure...)

File: docs/community/README.md (hunk 5 above)
Comment (evankanderson) [reply docs/community/README.md]: Good thought, done.

File: docs/community/README.md (hunk 6 above)
Comment (evankanderson) [reply docs/community/README.md:1]: Fixed this in `nav.yml`.

File: docs/community/contributing.md (hunk 8 above)
Comment (evankanderson) [reply docs/community/contributing.md]: Done

File: docs/community/governance.md (hunk 7 above)
Comment (evankanderson) [reply docs/community/governance.md:3]: Updated the title on this doc, as the nav title was better.

File: docs/community/README.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+* [Eventing](https://github.com/knative/eventing/issues/new?template=bug.md)
+* [Functions](https://github.com/knative/func/issues/new)
+
+A good bug report should include:
+
+- What you were trying to do, and what happened
+- What version of Knative and Kubernetes you are using (if using a cloud provider, indicate which one)
+- Relevant resource yaml, HTTP requests, or log lines
+
+## Community Meetups
```
Comment (psschwei) [inline docs/community/README.md:58]: just noticed we use basically the same text here and on the contributing page... we should probably create a snippet so as not to repeat ourselves.

That would make a good first issue, so I think we can skip updating now and I'll open an issue for it

**Review by psschwei — COMMENTED** [review]
This is a LGTM from me

Since you had done this in the other PR 
/assign @nainaz 

we should plan on merging this before we cut the docs for the 1.9 release

File: docs/community/README.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+
+Catch up with past community meetups on our [YouTube channel](https://www.youtube.com/playlist?list=PLQjzPfIiEQLLyCyLBKLlwDLfE_A-P7nyg).
+
+Stay tuned for new events by subscribing to the
+[calendar](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com&ctz=America%2FLos_Angeles) ([iCal export file](https://calendar.google.com/calendar/ical/knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com/public/basic.ics))
+and following us on [Twitter](https://twitter.com/KnativeProject).
+
+## Communication Channels
+
+Much of the community meets on [the Knative Slack](https://slack.knative.dev/).
```
Comment (psschwei) [inline docs/community/README.md:71]: now that we've moved to the CNCF slack, does this need to be updated?

Comment (psschwei) [reply docs/community/README.md:71]:
```suggestion
Much of the community meets on [the CNCF Slack](https://cloud-native.slack.com/) in the `#knative` channel.
```

**Review by psschwei — COMMENTED** [review]
/lgtm
/approve

File: docs/community/README.md (hunk 11 above)
Comment (psschwei) [reply docs/community/README.md:71]: we're going to do a separate PR for all the slack stuff

**Pull request merged**